lib/common/constants.js

```diff
@@ -26,7 +26,7 @@ function constantsFactory (path) {
               error: 'red',
               warning: 'yellow',
               info: 'green',
-              debug: 'blue',
+              debug: 'cyan',
```

Comment:
@yyscamper no not at all, this change is fine. I happen to be colorblind, so @heckj was just making sure I could actually tell the difference between cyan and green!